browser.js

```diff
@@ -64,11 +64,31 @@ function init() {
   backButton();
 }
 
+function setDarkMode() {
+	document.documentElement.classList.toggle('dark-mode', config.get('darkMode'));
+    if (config.get('darkMode')) {
```

Comment:
According to https://github.com/sindresorhus/caprine/blob/master/browser.js#L180-L181, it is to prevent "flashes" of the wrong color, which I believe sometimes occurs still. Although I think you can just have in the css a `background-color` rule...